src/components/FilePreviewer.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import FileIcon from './FileIcon';
+import LoadingIndicator from './LoadingIndicator';
+import type { FileUpload } from '../types';
```

Comment:
not a huge deal in this repo, but just to get in the habit should prob organize the imports like we do in react now, with newlines between each different import grouping

1) third party deps (react always first)
2) local imports grouped by depth and abc'd
3) type imports grouped by depth and abc'd